src/data/extract_census_tracts_json.py

```diff
@@ -0,0 +1,153 @@
+#    Copyright 2016-2018 SAS Project Authors. All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+"""extract_census_tracts utility to download and convert the census_tract and split to single file.
+
+This extract_census_tracts_json.py will help to perform following operations:
+1.Download original census tract data from the USGS web site  by providing the
+ command line option --retrieve
+2.Convert the shape file to GeoJSON by providing the command line option --convert
+3.Split the GeoJSON file into individual census tract files by providing the command line
+ option --split
+'Spectrum-Access-System/data/census' directory will be created/used to download original
+ census tract data and convert the shape file to geojson for the option #1 and #2.
+'Spectrum-Access-System/data/census_tracts' directory will be created/used to split the
+ geojson file into individual census_tract by prefix GEOID as file name.
+
+"""
+import argparse
+import os
+import re
+import sys
+import glob
+import json
+import ftputil
+from reference_models.geo import census_tract
+
```

Comment:
See my comment on the driver file about moving the processing function within this script file.
They are only used when creating the database, so best to keep them localized here.

Reply:
Fixed   as per suggestion.